scripts/mktcli.js

```diff
@@ -48,23 +59,47 @@ const main = {
       )
 
     program
-      .command('land-owner <coord>')
-      .description('Get the land owner of a (x,y) coordinate')
+      .command('asset-owner <assetId> <assetType>')
+      .description('Get the owner of an asset')
       .action(
-        asSafeAction(async coord => {
-          const [x, y] = parseCLICoords(coord)
-          const contract = eth.getContract('LANDRegistry')
-          const owner = await contract.ownerOfLand(x, y)
+        asSafeAction(async (assetId, assetType) => {
+          const asset = await getAssetFromCLIArgs(assetId, assetType)
+          let contract
+
+          switch (assetType) {
+            case ASSET_TYPES.parcel: {
+              contract = eth.getContract('LANDRegistry')
+              break
+            }
+            case ASSET_TYPES.estate: {
+              contract = eth.getContract('EstateRegistry')
+              break
+            }
+          }
 
-          const parcel = await Parcel.findOne({ x, y })
-          const dbOwner = parcel.owner || parcel.district_id || 'empty'
+          const owner = await contract.ownerOf(asset.token_id)
+          const dbOwner = asset.owner || asset.district_id || 'empty'
 
-          log.info(`(land-owner) coords:(${x},${y})`)
+          log.info(`(asset-owner) id:(${asset.id})`)
           log.info(`blockchain => ${owner}`)
           log.info(`db         => ${dbOwner}`)
         })
       )
 
+    program
+      .command('asset-update-operator <assetId> <assetType>')
+      .description('Get the update operator of an asset')
+      .action(
+        asSafeAction(async (assetId, assetType) => {
+          const asset = await getAssetFromCLIArgs(assetId, assetType)
+          const contract = eth.getContract('LANDRegistry')
```

Comment:
We should have this command for Estates too.

Related to [have a helper function to get the contract based on the assetType](https://github.com/decentraland/marketplace/pull/588/files#diff-14c145377d6748040b73ee576c8e1d57R69)